index.js

```diff
@@ -9,6 +9,10 @@ module.exports = function sanitizer_plugin(md, options) {
       // <a href="url" title="(optional)"></a>
       patternLinkOpen = '<a\\s([^<>]*href="[^"<>]*"[^<>]*)\\s?>',
       regexpLinkOpen = RegExp(patternLinkOpen, 'i'),
+      // <mark class="classname" js-search-message-id="id">
+      patternMarkdown =
+        '<mark\\sclass="([\\w\\-\\_\\s]*)"\\sjs-search-message-id="(\\w*)">'
```

Comment:
I think we don't need to whitelist the `js-search-message-id` attribute because we've already moved it to `StackMessage` component.

Reply:
I'm thinking about undoing all the changes we made in [PR](https://github.com/taskworld/tw-frontend/pull/10617) . Otherwise, we would still need to nest tags inside

Reply:
Ah I see. that's fine.